mozilla_django_oidc/auth.py

```diff
@@ -126,7 +126,9 @@ def authenticate(self, **kwargs):
         response = requests.post(self.OIDC_OP_TOKEN_ENDPOINT,
                                  data=token_payload,
                                  verify=import_from_settings('OIDC_VERIFY_SSL', True))
-        response.raise_for_status()
+
+        if not response.status_code == requests.codes.ok:
+            return None
```

Comment:
Also, just so I don't forget, any kind of message back to the user requires localization and might be something the site wants to personalize itself.